### DISCUSSION TITLE
Would it be possible to have auto updates?

### DISCUSSION THREAD

**Discussion by paulusgi** [seerge/g-helper#216] [Ideas]
An option that can be enabled or not to auto update the app or just a button to click and update. (without downloading it manually)

**Comment by seerge** [comment]
@paulusgi app already has update checker, version label becomes red and links to download page (it checks only for "latest" releases, and skips "pre-releases")

**Reply by paulusgi** [reply]
¡Oh! I know, maybe I didn’t explain it well. I mean a feature that allows the user to update the application without having to download it manually using a web browser. An integrated downloader to automatically download the latest version and replace the old G-Helper.exe file.

**Reply by seerge** [reply]
@paulusgi ah, that one is actually too much hassle for me to implement (by hand). And if I will bundle app with some installer, it will become bigger and heavier, so I'm not a big fan of that :)

**Reply by paulusgi** [reply]
True, I understand. I love the lightweight of the app too. I had this idea so that it could be updated by itself more easily once installed. But well... it is not so difficult to download and replace. But people tend to be very vague and many don’t even know how to download and replace files. Anyway, as this now works for me perfectly which is what really matters. Thank you very much.
